README.md

```diff
@@ -74,6 +75,7 @@ You are responsible for the cost of the AWS services used while running this sam
 1. On the "*Specify stack details*" screen you may customize the following parameters of the CloudFormation stack:
    * **Stack Name:** (Default: EngagementMeter) This is the name that is used to refer to this stack in CloudFormation once deployed. The value must be 15 characters or less.
    * **CollectionId:** (Default: RekogDemo) AWS Resources are named based on the value of this parameter. You must customise this if you are launching more than one instance of the stack within the same account.
+   * **IncludeCloudFrontDistribution**  (Default: false) The Web UI needs to be publicly accessible. If you have S3 public ACLs blocked on an account level, you can enable CloudFront to circumvent that. **Note:** including a CloudFront distribution will significantly increase the deploy time (from less than 5m to 30m+)
```

Comment:
```suggestion
   * **IncludeCloudFrontDistribution**  (Default: false) Creates a CloudFront distribution for accessing the web interface of the demo. This must be enabled if S3 Block Public Access is enabled at an account level. **Note:** Creating a CloudFront distribution may significantly increase the deploy time (from approximately 5 minutes to over 30 minutes)
```

Whilst the phrase "circumvent" is technically correct, my feel is there's a tone of deception to it, so I think it's best to avoid.